Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -380,6 +380,48 @@ namespace AZ
             }
         }
 
+        void MeshFeatureProcessor::SetMinimumScreenCoverage( const MeshHandle& meshHandle, float minimumScreenCoverage)
```

Comment:
spacing

Reply:
I'm having trouble seeing what needs different spacing on the line above. Is it a spaces tabs thing?

Reply:
wait, I'm sorry, I see what you mean. The space before const,.